pages/linux/parted.md

```diff
@@ -0,0 +1,36 @@
+# parted
+
+> A partition manipulation program.
+> More information: <https://www.gnu.org/software/parted/parted.html>.
+
+- List partitions on all block devices:
+
+`sudo parted --list`
+
+- Start to manipulate disk partition:
+
+`sudo parted {{/dev/sdX}}`
+
+- Create a new partition table of label-type directly, label-type can be gpt, msdos etc:
+
+`sudo parted --script {{/dev/sdX}} mklabel {{gpt}}`
+
+- Show disk partition information in interactive mode:
+
+`print`
+
+- Select a disk in interactive mode:
+
+`select {{/dev/sdX}}`
+
+- Create a 16GB part-type partition for filesystem type in interactive mode, (part-type should be one of "primary", "logical", or "extended"):
+
+`mkpart {{part-type}} {{filesystem-type}} {{0%}} {{16G}}`
```

Comment:
Generally we prefer underscores in tldr pages :-) Also, maybe this would read better?

```suggestion
- Interactively create a 16GB partition with a given filesystem:

`mkpart {{primary|logical|extended}} {{filesystem}} {{0%}} {{16G}}`
```